src/edu/psu/sweng500/team8/gui/SudokuGUI.java

```diff
@@ -346,12 +382,162 @@ private void radHardActionPerformed(java.awt.event.ActionEvent evt) {// GEN-FIRS
 	private void jButton11ActionPerformed(java.awt.event.ActionEvent evt) {// GEN-FIRST:event_jButton11ActionPerformed
 		// TODO add your handling code here:
 		// Create a file chooser
+		
 		final JFileChooser fc = new JFileChooser();
-		// In response to a button click:
+		fc.setFileFilter(new FileFilter()
+        {
+           @Override
+           public boolean accept(File file)
+           {
+              return (file.getName().toUpperCase().endsWith(".SUDOKU") || file.isDirectory());
```

Comment:
It shows the directories which can be opened, so we can operate on the files inside. If we don't accept directories only the files are shown.